app/src/main/res/values/colors.xml

```diff
@@ -3,5 +3,8 @@
   <color name="colorPrimary">#008577</color>
   <color name="colorPrimaryDark">#00574B</color>
   <color name="colorAccent">#D81B60</color>
+  <color name="colorIvory">#FFFFF0</color>
```

Comment:
Let's keep colors contextual. I believe we're using this as the concept card dialog background color.